index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html>
+
+    <head>
+        <title>Fakturomania</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.2/angular.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.2/angular-route.js"></script>
+        <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+        <script src="js/ui-bootstrap.js"></script>
+        <script src="js/index.js"></script>
+        <script src="js/menu.component.js"></script>
+
+        <script src="js/company/company.js"></script>
+        <script src="js/company/companyRepository.js"></script>
+        <script src="js/company/companyController.js"></script>
+
+        <script src="js/product/product.js"></script>
+        <script src="js/product/productRepository.js"></script>
+        <script src="js/product/productController.js"></script>
+
+        <script src="js/invoice/invoice.js"></script>
+
+    </head>
+
+    <body ng-app="firmApp">
+
+       <div class="nav">
+           <menu></menu>
+       </div>
+
+       <div ng-view></div>
+
+    </body>
+</html>
```

Comment:
teraz jak wyrzuciles menu do komponentu, i zrobiles poprawny routing, ten plik jest prawie pusty - tak powinno byc! bardzo ladnie to wyglada.